Readme.md

```diff
@@ -1,64 +1,114 @@
+experience will speed up. Scalar sets advanced Git config settings,
+maintains your repositories in the background, and helps reduce data sent
+across the network.
 
-See [the documentation](docs/index.md) for more details.
+You can learn more about Scalar in this video: [Git at Scale for Everyone](https://www.youtube.com/watch?v=USLB1gwl1vA).
 
 ## Table of Contents
 
-* [Installing on macOS](#installing-on-macos)
-* [Installing on Windows](#installing-on-windows)
-* [Quick start](#quick-start)
+* [Getting Started](#getting-started)
+* [Example Workflow](#example-workflow)
 * [License](#license)
+* [Code of Conduct](#code-of-conduct)
+
+## Getting Started
+
+Full details can be found in [our documentation page](https://github.com/microsoft/scalar/blob/HEAD/docs/index.md).
+
+### Installing on macOS
+
+Scalar is installed and updated on macOS via [Homebrew](https://brew.sh/).
+
+```sh
+brew tap microsoft/git
+brew cask install scalar
+```
+
+If you wish to use the [GVFS Protocol][gvfs-protocol], then you will
+instead need the cask that uses [our custom fork of Git][microsoft-git]:
+
+```sh
+brew tap microsoft/git
+brew cask install scalar-azrepos
+```
+
+When new versions of Scalar are available, you can upgrade in a few
+different ways. First, you can use `brew`:
+
+```sh
+brew update
+brew upgrade --cask scalar[-azrepos]
+```
+
+Alternatively, you can run `scalar upgrade` and it will run the necessary
+`brew` commands on your behalf.
+
+If your repository has many files in the working directory, then you might
+want to install [Watchman](https://github.com/facebook/watchman), which
+Scalar will detect and configure with Git's File System Monitor feature.
+
+```sh
+brew install watchman
+```
+
+### Installing on Windows
 
-Installing on macOS
-------------------
+To install Scalar on Windows, download `Installers_Windows_Release.zip`
+from the [latest release](https://github.com/microsoft/scalar/releases)
+and extract all of the files.  Run the contained installers for Git and
+Scalar, with names matching these patterns:
 
-To install Scalar on macOS,
-[download the `Installers_macOS_Release.zip` from the releases page](https://github.com/microsoft/scalar/releases).
-Extract the `Installers_macOS_Release` folder, `cd` into it, and run `./InstallScalar.sh` in a Terminal window.
-The script may prompt for your password as it installs the following components:
+ * `Git\Git-2.XX.Y-vfs.*.*-64-bit.exe`
+ * `Scalar\SetupScalar.YY.MM.<sprint>.<minor>.exe`
 
-* [Git](https://github.com/microsoft/git) (with custom patches)
-* [Git Credential Manager Core](https://github.com/microsoft/Git-Credential-Manager-Core)
-* Scalar
-* [Watchman](https://github.com/facebook/watchman), unless you use the `--no-watchman` argument.
+_Note: we are investigating package management tools to have the Windows
+install process be as simple as the `brew` process on macOS. See
+[#433](https://github.com/microsoft/scalar/issues/433) for progress on
+this feature._
 
```

Comment:
Do we want a ### Installing on Linux -- coming soon section ?

Reply:
I'll add the section when we have something to show for it ;)

That might include an update when we have an install-from-source option, replaced by "Download `.deb` from releases", then replaced by "use `apt-get`".